README.md

```diff
@@ -599,6 +600,212 @@ cvvElement.mount('#cvv'); //Assumes there is a div with id='#cvv' in the webpage
+ format: String, // Optional, format for the element (only applicable currently for EXPIRATION_DATE ElementType).
+ enableCopy: false, // Optional, enables the copy icon in collect and reveal elements to copy text to clipboard. Defaults to 'false').
+};
+const element = container.create(collectElement, options);
+```
+The `table` and `column1` fields indicate which table and column the Element corresponds to.
+
+skyflowID indicates the record that you want to update.
+
+**Notes:** 
+- Use dot-delimited strings to specify columns nested inside JSON fields (for example, `address.street.line1`)
+
+### Step 3: Mount Elements to the DOM 
+To specify where the Elements are rendered on your page, create placeholder `<div>` elements with unique `id` tags. For instance, the form below has three empty elements with unique IDs as placeholders for three Skyflow Elements.
+```html
+<form>
+ <div id="cardNumber" />
+ <br/>
+ <div id="expireDate" />
+ <br/>
+ <div id="cvv" />
+ <br/>
+ <button type="submit">Submit</button>
+</form>
+```
+Now, when you call the `mount(domElement)` method, the Elements is inserted in the specified divs. For instance, the call below inserts the Element into the div with the id "#cardNumber".
+```javascript
+element.mount('#cardNumber');
+```
+Use the `unmount` method to reset a Collect Element to its initial state.
+```javascript
+element.unmount();
+```
+
+
+### Step 4: Update data from Elements 
+When the form is ready to submit, call the `collect(options?)` method on the container object. The `options` parameter takes a object of optional parameters as shown below:
+- `tokens`: indicates whether tokens for the collected data should be returned or not. Defaults to 'true'
+- `additionalFields`: Non-PCI elements data to update or insert into the vault which should be in the records object format.
+- `upsert`: To support upsert operations while collecting data from Skyflow elements, pass the table and column marked as unique in the table.
+
+```javascript
+const options = {
+ tokens: true, // Optional, indicates whether tokens for the collected data should be returned. Defaults to 'true'.
+ additionalFields: {
+   records: [
+     {
+       table: "string", // Table into which record should be updated.
+       fields: {
+         column1: "value", // Column names should match vault column names.
+         skyflowID: "value", //  The skyflow_id of the record to be updated.
+         // ...additional fields here.
+       },
+     },
+     // ...additional records here.
+   ],
+ }, // Optional
+ upsert: [
+   // Upsert operations support in the vault
+   {
+     table: "string", // Table name
+     column: "value", // Unique column in the table
+   },
+ ], // Optional
+};
+container.collect(options);
+```
+**Note:** `skyflowID` is required if you want to update the data. If `skyflowID` isn't specified, the `collect(options?)` method creates a new record in the vault.
+
+### End to end example of collecting data with Skyflow Elements
```

Comment:
The title should be "Complete example of updating through Skyflow Elements"

Reply:
changes done